index.html

```diff
@@ -0,0 +1,72 @@
+<!DOCTYPE html>
+<html lang="en">
+
+<head>
+  <title>Apple</title>
+  <meta charset="UTF-8" />
+  <link rel="stylesheet" href="styles.css" />
+  <script src="https://use.fontawesome.com/729dceb01f.js"></script>
+</head>
+
+<body>
+  <main>
+    <nav>
+      <div class="nav-first">
+          <i class="fa fa-apple"> <a href=""></a></i>
+          <i><a href="#">Store</a></i>
+          <i><a href="#">Mac</a></i>
+          <i><a href="#">iPod</a></i>
+          <i><a href="#">iPhone</a></i>
+          <i><a href="#">iPad</a></i>
+          <i><a href="#">iTunes</a></i>
+          <i><a href="#">Support</a></i>
+          <i class="fa fa-search"><a href=""></a></i>
+      </div>
+      <div class="nav-last"></div>
+    </nav>
+    <div class="description">
+      <h1>What will your verse be?</h1>
+      <h2>
+        Everyone has something to share. See how people are using iPad in
+        their unique ways to add to the world's story.
+      </h2>
+      <h3>
+        Watch the <a href="">"Your Verse" </a> film and explore the stories
+        <i class="fa fa-play-circle"></i>
+      </h3>
+    </div>
+  </main>
+  <div class="promotions">
+      <div class="apple-images">
+        <a href=""><img src="images/image1.jpg" alt="" /></a> 
+        <a href=""><img src="images/image2.jpg" alt="" /></a> 
+        <a href=""><img src="images/image3.jpg" alt="" /></a> 
+        <a href=""><img src="images/image4.jpg" alt="" /></a> 
+      </div>
+    </div>
+  <div class="footer">
```

Comment:
Use more semantic HTML like using the <footer> tag instead of using divs with classes to mimic semantics